orchestration/scripts/move-error-logs-to-gcs.sh

```diff
@@ -0,0 +1,12 @@
+set -euo pipefail
+
+declare -r LOGS_DIR=../../logs
+declare -r LOG_FILE=${LOGS_DIR}/errors.log
+declare -r NEW_FILENAME=${LOGS_DIR}/${TIMESTAMP}.log
+
+# rename the file to the start time of the import
+mv ${LOG_FILE} ${NEW_FILENAME}
+# move the file to a bucket
+gsutil cp ${NEW_FILENAME} ${BUCKET_NAME}/errors/
+rm ${NEW_FILENAME}
+
```

Comment:
did you test out if this bash script/series of commands works in general? It looks fine to me, but just want to confirm

Reply:
Yeah, thanks for checking. I tested it using a bucket I had been using to test other things:
```
TIMESTAMP=01-01-2020 BUCKET_NAME=gs://monster-test-bucket/encode-explorer-backfill-requests bash -c './move-error-logs-to-gcs.sh'
```
And it worked (renamed and copied the file to an "errors" folder in the bucket, and then deleted the file locally).

Reply:
This reminded me of one thing that might be worth noting though - I had been running this script from inside the `/orchestration/scripts` directory, but the script doesn't work when you run it from a different directory because it uses a relative path to find the log file. Do you happen to know if there is a way to do this without using a relative path? Or if the relative path I have here would work with the way Argo runs things?

If not, we can see what happens when we run it this afternoon.